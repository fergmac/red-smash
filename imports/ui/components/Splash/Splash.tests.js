import Splash from './'
import React from 'react'
import { Meteor } from 'meteor/meteor'
import { shallow } from 'enzyme'
import { chai } from 'meteor/practicalmeteor:chai'

if (Meteor.isClient) {
  describe('component - Splash', () => {
    describe('render', () => {
      it('should render', () => {
        const item = shallow(<Splash />);
        chai.assert.equal(item.text(), 'REDSMASH');
      });
    });
  });
}