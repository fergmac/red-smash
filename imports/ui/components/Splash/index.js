import React, { Component } from 'react';
import Paper from 'material-ui/paper';

const styles = {
  giant: {
    fontSize: '20vw',
  },
}

export default class Splash extends Component {
  render() {
    return (
      <h1 style={styles.giant}>RED<br />SMASH</h1>
    );
  }
}
