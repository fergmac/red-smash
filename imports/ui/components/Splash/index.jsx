import React, { Component } from 'react';
import Paper from 'material-ui/paper';

const styles = {
  wrapper: {
    height: '100vh',
    // margin: '-70px -8px 0px -8px',
    margin: '-70px 0px 0px 0px',
    display: 'flex',
    flexFlow: 'column nowrap',
    justifyContent: 'center',
    alignItems: 'center'
  },
  giant: {
    fontSize: '20vw',
    // transform: 'rotate(45deg)'
  }
}

export default class Splash extends Component {
  render() {
    return (
      <div style={styles.wrapper}>
        <h1 style={styles.giant}>RED<br />SMASH</h1>
      </div>
    );
  }
}
