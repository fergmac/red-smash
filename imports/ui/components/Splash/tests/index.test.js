import React from 'react'
import Splash from '../'
import renderer from 'react-test-renderer'

describe('Splash', () => {
  it('renders a Splash')
})

test('renders the Splasher', () => {
  const tree = renderer.create(
    <Splash />
  ).toJSON()
  expect(tree).toMatchSnapshot()
})
