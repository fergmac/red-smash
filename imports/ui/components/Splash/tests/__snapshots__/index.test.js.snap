exports[`test renders the Splasher 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "display": "flex",
      "flexFlow": "column nowrap",
      "height": "500px",
      "justifyContent": "center",
    }
  }>
  <h1
    style={
      Object {
        "fontSize": "20vw",
      }
    }>
    RED
    <br />
    SMASH
  </h1>
</div>
`;
